app/controllers/guest_rooms_controller.rb

```diff
@@ -0,0 +1,15 @@
+class GuestRoomsController < ApplicationController
+
+  # def create
+  #   GuestRoom.create!(room_id: params[:room_id], guest_id: params[:id])
+  #   redirect_to "/guests/#{params[:id]}"
+  # end
+
+  def create
+    gr = GuestRoom.new(room_id: params[:room_id], guest_id: params[:id])
```

Comment:
Great work with this `GuestRoomsController`, it's exactly the kind of setup I would do. 🙌